src/v0/destinations/user/transform.js

```diff
@@ -135,16 +139,15 @@ const groupResponseBuilder = async (message, destination) => {
     );
     return responseBuilder(payload, endpoint, method, apiKey);
   }
-  throw new CustomError("[ User.com ] :: No user found with given userId", 400);
+  throw new InstrumentationError(
```

Comment:
Should it be `NetworkInstrumentationError` as this resulted after contacting the server?

Reply:
updated